doc/index.qdoc

```diff
@@ -0,0 +1,173 @@
+/****************************************************************************************
+**
+** Copyright (C) 2018 Jolla Ltd.
+** Contact: Chris Adams <chris.adams@jollamobile.com>
+** All rights reserved.
+**
+****************************************************************************************/
+
+/*!
+\contentspage {Sailfish OS Secrets and Crypto Framework Contents}
+\page index.html
+
+\title Secret Data and Cryptographic Operations in Sailfish OS
+
+Applications running on highly-connected devices require the capability to
+perform advanced cryptographic operations, in order to protect customer data,
```

Comment:
User data maybe? Being customer is only one aspect of our users.

Reply:
done